services/wallet/requests/router_input_params.go

```diff
@@ -55,6 +55,9 @@ type RouteInputParams struct {
 	PublicKey string       `json:"publicKey"`
 	PackID    *hexutil.Big `json:"packID"`
 
+	// Used internally
+	PathTxCustomParams map[string]*PathTxCustomParams `json:"-"`
```

Comment:
if used internally it could be lower case?
that way it won't even be exported in json

Reply:
It cannot cause it's a different package. So either this or to have a function that will set this property, but more or less the same thing.